pylab/website/tests/test_comments.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+
+from django.test import TestCase
+
+from django.contrib.auth.models import User
+
+from django_comments.models import Comment
+
+from pylab.core.models import Project
+
+
+class CommentsTests(TestCase):
+
```

Comment:
It would be better to use ```django_webtest.WebTest``` instead of ```django.test.TestCase```. You could use ```pylab.website.tests.test_projects.py``` as an example. ```WebTest``` interface is simpler.

You can test if value was saved to database correctly like this:
```
        self.assertEqual(
            list(Comment.objects.values_list('comment')),
            [('test comment')]
        )
```